_episodes/01-introduction.md

```diff
@@ -66,20 +68,27 @@ print(deltaG)
 ```
 {: .output}
 
-Each variable is some particular type of data.  The most common types of data are strings, integers, and floats or floating point numbers.  In the code block below `label` is a string and `DeltaG` is a float.  It also shows how you can print more than one thing.
+Each variable is some particular type of data.  The most common types of data are strings (`str`), integers (`int`), and floating point numbers (`float`).  You can identify the data type of any variable with the function `type(variable_name)`.
 ```
-label = 'Delta G'    #this is a string
-print(label, ':', deltaG)
+type(deltaG)
 ```
-{: .language-python}
-
+{: .language.python}
 ```
-Delta G : -3640.7000000000003
+float
 ```
 {: .output}
+You can change the data type of a variable like this.  This is called casting.
+```
+deltaG = str(deltaG)
+type(deltaG)
+```
+{: .language-python}
+```
+str
+```
 
 ## Lists and Slices
-Another common data structure in python is the list.  Lists can be used to group several values or variables together, and are declared using square brackets [ ]. List values are separated by commas. Python has several built in functions which can be used on lists. The built-in function `len` can be used to determine the length of a list.
+Another common data structure in python is the list.  Lists can be used to group several values or variables together, and are declared using square brackets [ ]. List values are separated by commas. Python has several built in functions which can be used on lists. The built-in function `len` can be used to determine the length of a list. This code block also demonstrates how to print multiple variables.
```

Comment:
The example is only printing one stored variable. I'm guessing you mean the combination of the string 'The length..." and  the variable `length`, but I think this phrasing could be confusing.